packages/page-staking/src/ParachainStakingApp/index.tsx

```diff
@@ -0,0 +1,164 @@
+// Copyright 2017-2022 @polkadot/app-staking authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { AppProps, ThemeProps } from '@polkadot/react-components/types';
+
+import React, { useEffect, useMemo, useState } from 'react';
+import { Route, Switch } from 'react-router-dom';
+import styled from 'styled-components';
+
+import { Tabs } from '@polkadot/react-components';
+import { useApi, useBestNumber, useCall } from '@polkadot/react-hooks';
+import { BN } from '@polkadot/util';
+
+import { useTranslation } from '../translate';
+import CollatorList from './CollatorList';
+import Summary from './Summary';
+import { ApiResult, CollatorState, CollatorStateRaw, OwnerAmount, RoundInfo } from './types';
+import UserDelegations from './UserDelegations';
+
+function ParachainStakingApp ({ basePath, className = '' }: AppProps): React.ReactElement<AppProps> {
+  const { api } = useApi();
+  const { t } = useTranslation();
+
+  // summary info
+  const roundInfo = useCall<RoundInfo<unknown>>(api.query.parachainStaking.round);
+  const totalSelected = Number(useCall<string>(api.query.parachainStaking.totalSelected));
+  const totalSelectedStaked = (useCall<BN>(api.query.parachainStaking.staked, [roundInfo?.current]));
+  const inflation = (useCall<{annual: {ideal: ApiResult}}|undefined>(api.query.parachainStaking.inflationConfig));
+  const inflationPrct = inflation?.annual.ideal.toHuman();
+  const parachainBondInfo = (useCall<{percent: ApiResult}|undefined>(api.query.parachainStaking.parachainBondInfo));
+  const parachainBondInfoPrct = parachainBondInfo?.percent.toHuman();
+  const bestNumberFinalized = useBestNumber();
+  const collatorCommission = (useCall<ApiResult|undefined>(api.query.parachainStaking.collatorCommission));
+  // Fetch all collator states using entries
+  const allCollators = useCall<[Uint8Array, CollatorStateRaw][]>((api.query.parachainStaking.candidateState).entries, []);
```

Comment:
@jacogr how do I make sure this information stays up to date without refreshing the page?